dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/SourcesToStartingPoints.scala

```diff
@@ -43,7 +42,7 @@ class SourceTravsToStartingPointsTask[NodeType](sourceTravs: Traversal[NodeType]
     val sources: List[StoredNode] = sourceTravs
       .flatMap(_.toList)
       .collect { case n: StoredNode => n }
-      .dedup
+      .distinct
```

Comment:
Looks like we're breaking all queries containing `dedup` here. Wondering if we could just introduce an alias.